src/Command/InfectionCommand.php

```diff
@@ -324,6 +328,61 @@ protected function initialize(InputInterface $input, OutputInterface $output): v
         $this->eventDispatcher = $this->container['dispatcher'];
     }
 
+    private function initContainer(InputInterface $input): void
```

Comment:
Disclaimer: I'm not a big fan of this code, but IMO it makes more sense that way. Indeed those parameters are defined here in `InfectionCommand`. So normalizing and interpreting those values should be the responsibility of the command, not the container.

If you are worried this makes the command needlessly bloated: don't worry, I got your back. In the future we want to make the command slimmer and, ideally, reach the point where it's only about interpreting the arguments & options and triggering the appropriate service with the right value. When this stage is reached, the command should look way slimmer than it is now. So fear not, this "bloated" state is only temporary.